examples/MDF/README.md

```diff
@@ -1,64 +1,81 @@
 # MDF Examples
 
-Examples of Python, JSON and YAML files to illustrate the structure and usage of MDF.
+Examples of [Python](https://fileinfo.com/extension/py), [JSON](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/JSON#:~:text=JavaScript%20Object%20Notation%20(JSON)%20is,page%2C%20or%20vice%20versa) and [YAML](https://circleci.com/blog/what-is-yaml-a-beginner-s-guide/) files to illustrate the structure and usage of MDF.
 
-[Simple](#simple-example) | [ABCD](#abcd) | [Arrays](#arrays) | [States](#states) | [Conditions](#conditions) | [Parameters and Functions](#parameters-and-functions)
+<a href="#simple"> Simple </a>| <a href="#abcd"> ABCD </a> | <a href="#arrays"> Arrays </a> | <a href="#st"> States </a> | <a href="#conditions"> Conditions </a> | <a href="#parameters-and-functions"> Parameters and Functions </a>
+
+<p id="simple"></p>
 
 ## Simple example
 
-[Python source](simple.py) | [JSON](Simple.json) | [YAML](Simple.yaml)
+[Python Source](https://github.com/ModECI/MDF/blob/update_readme_links/examples/MDF/simple.py) | [JSON](https://github.com/ModECI/MDF/blob/main/examples/MDF/Simple.json) | [YAML](https://github.com/ModECI/MDF/blob/main/examples/MDF/Simple.yaml)
 
 A simple example with 2 [Nodes](https://mdf.readthedocs.io/en/latest/api/Specification.html#node) connected by an [Edge](https://mdf.readthedocs.io/en/latest/api/Specification.html#edge):
 
-![simple](images/simple.png)
+<img src="https://raw.githubusercontent.com/ModECI/MDF/main/examples/MDF/images/simple.png" width="165" height="135" />
```

Comment:
Looks good in the generated html @kusanele, but double check how it looks in the markdown:
https://github.com/ModECI/MDF/blob/c3fda2e94b9585b407f29a3078530ab6f3925ae8/examples/MDF/README.md
<img width="670" alt="Screenshot 2023-01-13 at 17 43 40" src="https://user-images.githubusercontent.com/1556687/212386540-2d053d27-0af1-4150-a2eb-86cf26d04488.png">